internal/olm/manager.go

```diff
@@ -82,7 +79,7 @@ func (m *Manager) Install() error {
 		return err
 	}
 
-	log.Infof("Successfully installed OLM version %q", m.Version)
+	log.Infof("Successfully installed OLM")
```

Comment:
I'm probably missing something important, but is there a reason we need to remove this extra context about the version number that was installed?